components/Member/MySessions/Current.js

```diff
@@ -98,10 +98,12 @@ const UPDATE_SESSION = gql`
   mutation updateSession($sessionId: ID!, $session: SessionUpdateInput!) {
     sessions {
       session(id: $sessionId) {
-        update(session: $session) {
-          id
-          title
-          status
+        update {
```

Comment:
why is the double nested update necessary?

Reply:
The update() mutation is now deprecated and here for backward compatibility.